build.gradle

```diff
@@ -1,38 +1,33 @@
 plugins {
     id 'groovy'
     id 'java-gradle-plugin'
-    id 'com.gradle.plugin-publish' version "0.16.0"
+    id 'com.gradle.plugin-publish' version "1.2.2"
     id 'maven-publish'
     id 'idea'
     id 'eclipse'
-    id 'org.ajoberstar.reckon' version '0.12.0'
-    id 'org.ajoberstar.git-publish' version '2.1.3'
+    id 'org.ajoberstar.reckon' version '0.18.3'
+    id 'org.ajoberstar.git-publish' version '4.2.2'
 }
 
 group = 'com.bmuschko'
 
 gradlePlugin {
-    plugins {
```

Comment:
Why this change?

Reply:
It was giving issue with earlier configuration, so just fixed the issues.